src/themes/light-theme-with-move.js

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import createTheme from './creator';
+import {primitives as lightThemePrimitives} from './light-theme-primitives';
+
+const primaryFontFamily =
+  'UberMoveText, "Open Sans", "Helvetica Neue", Helvetica, sans-serif';
+
+const secondaryFontFamily =
+  'UberMove, UberMoveText, "Open Sans", "Helvetica Neue", Helvetica, sans-serif';
+
+export const LightThemeMove = createTheme(
+  {
+    ...lightThemePrimitives,
+    primaryFontFamily,
```

Comment:
do we need to pass `secondaryFontFamily ` in here?

Reply:
we don't - the factory has no concept of the `secondaryFontFamily`